src/NHSD.BuyingCatalogue.Organisations.Api/Settings/SmtpSettings.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace NHSD.BuyingCatalogue.Organisations.Api.Settings
+{
+    /// <summary>
+    /// SMTP server settings.
+    /// </summary>
+    internal sealed class SmtpSettings
```

Comment:
Could the class `SmtpSettings` class be made immutable? Once you have created an instance of this class do the values want to change again?